examples/entry-list/src/locations/EntryList.js

```diff
@@ -0,0 +1,13 @@
+export const onEntryListUpdated = ({entries}) => {
```

Comment:
maybe let's make it `async` just to show that the listener can accept the promise as well, wdyt?